projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/videopress-value-section.tsx

```diff
@@ -0,0 +1,22 @@
+import type { FC } from 'react';
+
+import './style.scss';
+
+interface VideoPressValueSectionProps {
+	isPluginActive: boolean;
+	data: Window[ 'myJetpackInitialState' ][ 'videopress' ];
+}
+
+const VideoPressValueSection: FC< VideoPressValueSectionProps > = ( { isPluginActive, data } ) => {
+	if ( ! isPluginActive && data.videoCount ) {
+		return (
+			<div className={ 'videopress-card__video-count' }>
+				<span>{ data.videoCount }</span>
```

Comment:
Do we need `<span>` tag here?

Reply:
Hmmm I prefer it rather than just having text straight in the `<div>` element 🤔 Does that logic sound decent to you?

Reply:
Why cannot we have `<span className="videopress-card__video-count">`?

The logic sounds good to me, thanks for explaining!

Reply:
That could work style-wise 🤔 

I think I just like container enclosed items such as this. But I think I agree with you that it is not necessary in this case